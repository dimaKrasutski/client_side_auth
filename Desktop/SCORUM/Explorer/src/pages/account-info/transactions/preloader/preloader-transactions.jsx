import React, { Fragment } from 'react';

export const PreloaderRow = () => (
  <tr className="preloader-transactions preloader-row">
    <td>
      <div className="preloader" />
    </td>
    <td>
      <div className="preloader" />
    </td>
    <td>
      <div className="preloader" />
    </td>
    <td>
      <div className="preloader float-right" />
    </td>
  </tr>
);

export const PreloaderTransactions = () => (
  <Fragment>
    <PreloaderRow />
    <PreloaderRow />
    <PreloaderRow />
  </Fragment>
);
