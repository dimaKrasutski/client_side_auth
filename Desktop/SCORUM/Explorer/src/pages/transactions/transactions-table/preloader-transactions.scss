.preloader-transactions {
  .preloader-hash {
    width: 100%;
  }

  .preloader-username {
    width: 50px;
  }
}
