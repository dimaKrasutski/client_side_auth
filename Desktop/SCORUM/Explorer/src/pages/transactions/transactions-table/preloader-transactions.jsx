import React, { Fragment } from 'react';
import './preloader-transactions.scss';

export const PreloaderRow = () => (
  <tr className="preloader-transactions preloader-row">
    <td>
      <div className="preloader preloader-hash" />
    </td>
    <td>
      <div className="preloader preloader-username" />
    </td>
    <td />
    <td>
      <div className="preloader preloader-username" />
    </td>
    <td>
      <div className="preloader preloader-sm" />
    </td>
    <td>
      <div className="preloader float-right" />
    </td>
  </tr>
);

export const PreloaderTransactions = () => (
  <Fragment>
    <PreloaderRow />
    <PreloaderRow />
    <PreloaderRow />
  </Fragment>
);
