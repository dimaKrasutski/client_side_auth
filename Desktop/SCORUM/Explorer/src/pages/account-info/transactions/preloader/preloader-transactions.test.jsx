import React from 'react';
import { PreloaderTransactions, PreloaderRow } from './preloader-transactions';

describe('<PreloaderTransactions />', () => {
  test('should match snapshot', () => {
    const wrapper = shallow(<PreloaderTransactions />);
    expect(wrapper).toMatchSnapshot();
  });
});

describe('<PreloaderRow />', () => {
  test('should match snapshot', () => {
    const wrapper = shallow(<PreloaderRow />);
    expect(wrapper).toMatchSnapshot();
  });
});
