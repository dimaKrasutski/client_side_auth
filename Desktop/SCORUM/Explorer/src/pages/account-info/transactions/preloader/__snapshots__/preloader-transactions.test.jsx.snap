// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<PreloaderRow /> should match snapshot 1`] = `
<tr
  className="preloader-transactions preloader-row"
>
  <td>
    <div
      className="preloader"
    />
  </td>
  <td>
    <div
      className="preloader"
    />
  </td>
  <td>
    <div
      className="preloader"
    />
  </td>
  <td>
    <div
      className="preloader float-right"
    />
  </td>
</tr>
`;

exports[`<PreloaderTransactions /> should match snapshot 1`] = `
<React.Fragment>
  <PreloaderRow />
  <PreloaderRow />
  <PreloaderRow />
</React.Fragment>
`;
